File: P14_2703/Klargest.py
arr = list(map(int, input().split()))
k = int(input())

index = 0
maxx = 0
for i in range(k):
    maxx = 0
    index = 0
    for j in range(len(arr)):
        if maxx < arr[j]:
            maxx = arr[j]
            index = j

    print(maxx, end=" ")
    arr[index] = 0

File: P14_2703/Klargest.c
#include <stdio.h>

int main()
{
    int n;
    printf("Enter no. of elements: ");
    scanf("%d", &n);

    int arr[n];

    for (int i = 0; i < n; i++)
        scanf("%d", &arr[i]);

    int k;
    printf("\nEnter no of largest elements: ");
    scanf("%d", &k);

    int maxx = 0;
    int j = 0;
    for (int i = 0; i < k; i++)
    {
        maxx = 0;
        j = 0;
        for (int i = 0; i < n; i++)
        {
            if (maxx < arr[i])
            {
                maxx = arr[i];
                j = i;
            }
        }

        printf("%d ", maxx);
        arr[j] = 0;
    }
}